brain-bar/build-app.sh

```
@@ -89,6 +89,170 @@ sanitize_branch_name() {
+            HEAD\ *)
+                head_sha="${line#HEAD }"
+                ;;
+            branch\ refs/heads/*)
+                local branch
+                branch="${line#branch refs/heads/}"
+                if [ "$(sanitize_branch_name "$branch")" = "$safe_branch" ]; then
+                    return 0
+                fi
+                ;;
+            "")
+                if [ -n "$detached_sha" ] && [ "${head_sha#$detached_sha}" != "$head_sha" ]; then
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)